packages/@glimmer/syntax/tests/parser-node-test.ts

```diff
@@ -470,3 +470,15 @@ test("allow {{undefined}} to be passed as a param", function() {
     b.mustache(b.path('foo'), [b.undefined()])
   ]));
 });
+
+test("Handlebars partial should error", function(assert) {
+  assert.throws(() => {
+    parse("{{> foo}}");
+  }, Error(`Handlebars partials are not supported: "{{> foo" on line "1:0"`));
```

Comment:
I was copying https://github.com/tildeio/glimmer/pull/397/files#diff-96d3a15c8247549ce9be44ee089dc869L132. I'll do `at L1:C2` if Ember has that.